app/models/digital_object/base.rb

```diff
@@ -21,7 +21,7 @@ class DigitalObject::Base
   
   def require_subclass_override!; raise 'This method must be overridden by a subclass'; end
   
-  def initialize()
+  def initialize(*args)
```

Comment:
I'm a little suspicious of the subclass part, but you can explicitly call
super with an empty argument. Fine by me.

On Mon, Dec 21, 2015 at 2:37 PM, Eric O <notifications@github.com> wrote:

> In app/models/digital_object/base.rb
> <https://github.com/cul/hyacinth/pull/8#discussion_r48181618>:
>
> > @@ -21,7 +21,7 @@ class DigitalObject::Base
> >
> >    def require_subclass_override!; raise 'This method must be overridden by a subclass'; end
> >
> > -  def initialize()
> > +  def initialize(*args)
>
> Do you mind if I just remove all cases where we pass the args hash? This
> class and its subclasses aren't supposed to accept any args in the
> initializer. I don't want the method signature to imply that they do.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cul/hyacinth/pull/8/files#r48181618>.
>